source/about/self-hosted-subscriptions.rst

```diff
@@ -99,13 +99,25 @@ To send this report, take a screenshot of **System Console > Site Statistics** a
 - We don't need your server address, so if it appears on your screenshot, you can redact it from the image.
 
 .. tip:: 
-   
-   Not sure where to take the screenshot? Please reach out to your account executive, Customer Success Manager, orders@mattermost.com, or support@mattermost.com for help.
-   
+
+   Not sure where to find the true-up review section? Please reach out to your account executive, Customer Success Manager, orders@mattermost.com, or support@mattermost.com for help.
+
 If you have more total active users than you purchased in your annual subscription, your Customer Success Manager will provide you with a true-up quote for the new users added.
 
 The additional invoice will be pro-rated based on the number of months left in your subscription term, including the months for the calendar quarter for the time you pull the report. We will not provide downward adjustments. Mattermost will invoice based on Mattermost’s `current list prices <https://mattermost.com/pricing/>`__.
 
+Quarterly true-up reports in version 7.9 or newer
+-------------------------------------------------
+
+Quarterly true-up reports have changed in Mattermost version 7.9.
+The reporting period is still the same, but we've introduced a process that is much easier than taking screenshots.
+
+To send the report, a System Admin must navigate to the **System Console** and open either the **Site Statistics** or **Team Statistics** pages.
+
+If the current date is within the True-Up reporting period, a panel will be visible at the top of the page with a button to share your systems statistics directly to Mattermost.
```

Comment:
How do you think we can make "True-up reporting period" clear here? We have the illustration, but it's currently in the "prior to version 7.9", which is also needed there.
<img width="980" alt="Screenshot 2023-01-31 at 6 40 02 PM" src="https://user-images.githubusercontent.com/101303489/215908639-0b3e3050-382a-4d5c-9561-c33cf863e985.png">

Reply:
I suppose two options:

1. Make a reference to the image, saying something like:
  - `If the current date is within the True-Up reporting period outlined in the previous section...`
2. Re-use the image in the new section

Thoughts?

Reply:
Ya re-using might be easier here